src/process.cr

```diff
@@ -47,8 +47,12 @@ class Process
     nil
   end
 
-  def self.ppid : LibC::PidT
```

Comment:
Ah, looks like you moved the `ppid` method in the file and the addition and deletion got split between two commits.